src/api/bitswap/stat.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+const moduleConfig = require('../../module-config')
+
+module.exports = (arg) => {
+  const send = moduleConfig(arg)
```

Comment:
Do this step at the `index.js` level. Otherwise, you are creating a 'send' instance for every API call.

And also, at the index.js, you should check if the `arg` is already a `send` instance, for when the user uses ipfs-api directly and not a subapi.